cirq/google/engine/engine_processor.py

```diff
@@ -277,29 +277,70 @@ def list_reservations(
 
     def get_schedule(
             self,
-            from_time: datetime.datetime = datetime.datetime.now(tz=utc),
-            to_time: datetime.datetime = datetime.datetime.now(tz=utc) +
-            datetime.timedelta(weeks=2),
-            time_slot_type: qenums.QuantumTimeSlot.TimeSlotType = None
+            from_time: Union[None, datetime.datetime, datetime.
+                             timedelta] = datetime.timedelta(),
+            to_time: Union[None, datetime.datetime, datetime.
```

Comment:
This parameter style should be kept in sync with list_reservations() just above.

Reply:
Done.